packages/react-core/src/components/Progress/Progress.tsx

```diff
@@ -34,6 +34,8 @@ export interface ProgressProps extends Omit<React.HTMLProps<HTMLDivElement>, 'si
   max?: number;
   /** Accessible text description of current progress value, for when value is not a percentage. Use with label. */
   valueText?: string;
+  /** Indicate whether to truncate the title */
+  isTruncated?: boolean;
```

Comment:
wdyt about calling this something like `isTitleTruncated` in case we add truncation to something else (like the label)? Then again we could always add a unique string to the other truncated thing (`isLabelTruncated`).

Reply:
@mcoker makes sense to me, thanks for the suggestion!  Pushed this update